### PR TITLE
Phylo Refactor

### PR DESCRIPTION
**Pull request by morsecodist**

### CODE REVIEW

File: short-read-mngs/Dockerfile [hunk 1]
```diff
@@ -138,5 +140,15 @@ ENV PATH="${PATH}:/rapsearch2/Src/"
 RUN apt-get purge -y g++ libperl4-corelibs-perl make
 WORKDIR /
 
+# For phylogenetic trees
+WORKDIR /tmp
+RUN wget https://sourceforge.net/projects/ksnp/files/kSNP3.1_Linux_package.zip
+RUN unzip -o kSNP3.1_Linux_package.zip
+WORKDIR /tmp/kSNP3.1_Linux_package/kSNP3
+RUN cp -r * /usr/local/bin/
+RUN sed -i 's:set kSNP=/usr/local/kSNP3:set kSNP=/usr/local/bin:g' /usr/local/bin/kSNP3 # edit the kSNP3 executable so it looks for other executables in /usr/local/bin
```
Comment (kislyuk) [inline short-read-mngs/Dockerfile:149]: Please move the comment to its own line for readability.

File: short-read-mngs/generate_phylo_tree.wdl [hunk 2]
```diff
@@ -0,0 +1,113 @@
+version development
+
+struct TaxonByteRange {
```
Comment (kislyuk) [inline short-read-mngs/generate_phylo_tree.wdl:3]: These structs are nice!

**Review by kislyuk — APPROVED** [review]
Please add a unit test like the one in https://github.com/chanzuckerberg/idseq-workflows/blob/main/tests/short-read-mngs/test_short_read_mngs.py (@mlin knows how to set it up if needed)

**Comment by morsecodist** [comment]
Closed in favor of https://github.com/chanzuckerberg/idseq-workflows/pull/104

**Comment by morsecodist** [comment]
Closed in favor of https://github.com/chanzuckerberg/idseq-workflows/pull/104

**Pull request closed without merging**